webapp/channels/src/components/admin_console/schema_admin_settings.tsx

```diff
@@ -505,8 +509,10 @@ export class SchemaAdminSettings extends React.PureComponent<Props, State> {
         }
 
         let inputType: 'text' | 'number' | 'textarea' = 'text';
+        let defaultValue: string | number = '';
         if (setting.type === Constants.SettingsTypes.TYPE_NUMBER) {
             inputType = 'number';
+            defaultValue = 0;
```

Comment:
For clarity, would it make sense to assign this to `setting.default`? Otherwise, this is sort of a `defaultDefaultValue` which can be hard to think through. (0/5)

Reply:
Maybe we could rename it to `zeroValue` instead since it's acting like zero values in Go